src/db/models/index.ts

```diff
@@ -1,5 +1,62 @@
-export * from './Cart.model';
-export * from './LoggedInUsers.model';
-export * from './Product.model';
-export * from './Token.model';
-export * from './User.model';
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-undef */
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
- [ ] Kindly remove this & solve the linter manually

Reply:
i've tried removing these and manually adding it in the eslint file but it doesn't work, that's why i add it in manually